README.md

```diff
@@ -4,33 +4,112 @@
 
 ## Terms and Conditions
 
-LAN Protocol. LiFi Labs, Inc. © 2015. All rights reserved. Usage of this documentation is bound by the [LIFX Developer Terms](http://www.lifx.com/pages/developer-terms-of-use).
+LAN Protocol. LiFi Labs, Inc. © 2021. All rights reserved. Usage of this documentation is bound by the [LIFX Developer Terms](http://www.lifx.com/pages/developer-terms-of-use).
 
-### Extended multizone
+### Available capabilities
 
-Multizone products like the LIFX Z and the LIFX Beam have an "Extended Multizone"
-api when you have new enough firmware. The version of firmware that enables this
-feature is specified by the `min_ext_mz_firmware` and
-`min_ext_mz_firmware_components` fields for a multizone product in products.json
+* `color` = The light changes physical appearance when the Hue value is changed
+* `chain` = The light may be connected to physically separated hardware
+  (currently only the LIFX Tile)
+* `matrix` = The light supports a 2D matrix of LEDs (the Tile and Candle)
+* `relays` = The device has relays for controlling physical power to something
+  (the LIFX Switch)
+* `buttons` = The device has physical buttons to press (the LIFX Switch)
+* `infrared` = The light supports emitting infrared light
+* `multizone` = The light supports a 1D linear array of LEDs (the Z and Beam)
+* `temperature_range` = An array of the minimum and maximum kelvin values this
+  device supports. If the numbers are the same then the device does not support
+  variable kelvin values. It is `null` for devices that don't have LEDs (the
+  LIFX Switch)
```

Comment:
It is 'null' for devices that aren't lighting products, i.e. don't have a kelvin range to set.  (Switch does have LEDs)

Reply:
reasonable technicality :) will fix

Reply:
Would it make sense to add HEV (i.e. Clean)?  It seems odd that we have unique product IDs for infrared products and still have it as a feature.  I think we should do the same for Clean then.

Reply:
we haven't made that part of the protocol public yet.

Reply:
actually, it seems I already had hev in here, so I've [added](https://github.com/LIFX/products/commit/999067e14a68a1287644cef1208cd4558ca1027c) it to the defaults now too. oops.